include/sys/time.h

```diff
@@ -46,6 +46,11 @@ typedef struct bintime {
     .frac = (uint64_t)((fp - __builtin_floor(fp)) * (1ULL << 63) * 2)          \
   }
 
+#define BINTIME2(s, fp)                                                        \
```

Comment:
This name doesn't look good for me.

If I understand correctly you introduce `BINTIME2` only because clang doesn't support static declaration combined with `__builtin_floor` in tests. If you doesn't plan to use it in other places maybe it will be good idea to move this macro into dedicated tests or change something other in tests?